tests/Functional/GetAssetsTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+declare(strict_types=1);
+
+namespace PayCrypto\Tests;
+
+use PHPUnit\Framework\TestCase;
+use PayCrypto\Resource\GetAssets;
+use PayCrypto\Collection\AssetsRecord;
+use ScriptFUSION\Porter\Specification\ImportSpecification;
+
+final class GetAssetsTest extends TestCase
+{
+    /** @var $apiKey This is the Coin API Key for test environment*/
+    private $apiKey = '4E861687-19D6-4894-87B9-E785B1EE3900';
```

Comment:
As mentioned before, you shouldn't be committing keys to your repository. They can be used by anyone, which is a security problem. Pass keys at invocation time via environment variables instead. This also fixes the duplication issue.